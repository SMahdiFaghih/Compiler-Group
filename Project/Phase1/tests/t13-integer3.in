12345678901234567890
01234567890123456789
