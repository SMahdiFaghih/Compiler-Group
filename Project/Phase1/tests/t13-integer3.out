T_INTLITERAL 12345678901234567890
T_INTLITERAL 01234567890123456789
